<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LuWPS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>标准</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号●</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号○</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号□</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(1,2,3)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(A,B,C)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(a,b,c)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(i,ii,iii)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(I,III,III)</string>
     </property>
    </item>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="printpreviewAction"/>
    <addaction name="printfaction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="UndoAction"/>
    <addaction name="RedoAction"/>
    <addaction name="CutAction"/>
    <addaction name="CopyAction"/>
    <addaction name="PasteAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menu_D">
     <property name="title">
      <string>字体(D)</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/img/character.png</normaloff>:/img/character.png</iconset>
     </property>
     <addaction name="boldAction"/>
     <addaction name="italicAction"/>
     <addaction name="underline"/>
    </widget>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>段落</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/img/paragraph.png</normaloff>:/img/paragraph.png</iconset>
     </property>
     <addaction name="leftAlignAction"/>
     <addaction name="centerAction"/>
     <addaction name="rightAlignAction"/>
     <addaction name="JustifyAction"/>
    </widget>
    <addaction name="menu_D"/>
    <addaction name="menu"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗体(&amp;W)</string>
    </property>
    <addaction name="closeaction"/>
    <addaction name="closeallaction"/>
    <addaction name="casecadeAction"/>
    <addaction name="Titleaction"/>
    <addaction name="previousaction"/>
    <addaction name="nextaction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutaction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="printfaction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="UndoAction"/>
   <addaction name="RedoAction"/>
   <addaction name="CutAction"/>
   <addaction name="CopyAction"/>
   <addaction name="PasteAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="boldAction"/>
   <addaction name="italicAction"/>
   <addaction name="underline"/>
   <addaction name="separator"/>
   <addaction name="leftAlignAction"/>
   <addaction name="centerAction"/>
   <addaction name="rightAlignAction"/>
   <addaction name="JustifyAction"/>
   <addaction name="colorAction"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="statusTip">
    <string>创建一个新文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/fileopen.png</normaloff>:/img/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="statusTip">
    <string>打开一个已存在的文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/saveasaction.png</normaloff>:/img/saveasaction.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A)</string>
   </property>
   <property name="statusTip">
    <string>以一个新名字保存文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="printpreviewAction">
   <property name="icon">
    <iconset>
     <normaloff>:/img/android-chrome-192x192.png</normaloff>:/img/android-chrome-192x192.png</iconset>
   </property>
   <property name="text">
    <string>打印预览...</string>
   </property>
   <property name="statusTip">
    <string>预览打印效果</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="printfaction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/print-fill.png</normaloff>:/img/print-fill.png</iconset>
   </property>
   <property name="text">
    <string>打印(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="UndoAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/undo.png</normaloff>:/img/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="statusTip">
    <string>撤销当前操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="RedoAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/redo2.png</normaloff>:/img/redo2.png</iconset>
   </property>
   <property name="text">
    <string>重做(R)</string>
   </property>
   <property name="statusTip">
    <string>恢复之前的操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="CutAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/cut.png</normaloff>:/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="statusTip">
    <string>重文档中剪切所选的内容，并放入剪切板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="CopyAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/copy.png</normaloff>:/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="statusTip">
    <string>复制所选的内容，并放入剪切板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="PasteAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/paste.png</normaloff>:/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="statusTip">
    <string>将剪切板的内容粘贴到文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/eglass-color.png</normaloff>:/img/eglass-color.png</iconset>
   </property>
   <property name="text">
    <string>颜色(C)...</string>
   </property>
   <property name="statusTip">
    <string>设置字体颜色</string>
   </property>
  </action>
  <action name="boldAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/bold.png</normaloff>:/img/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="italicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/italic.png</normaloff>:/img/italic.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="leftAlignAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/align-left.png</normaloff>:/img/align-left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐(L)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="centerAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/justify.png</normaloff>:/img/justify.png</iconset>
   </property>
   <property name="text">
    <string>居中(C)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字居中</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="rightAlignAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/align-right.png</normaloff>:/img/align-right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐(R)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字右对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="JustifyAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/two.png</normaloff>:/img/two.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐(J)</string>
   </property>
   <property name="statusTip">
    <string>将所选文字左右两端同时对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="closeaction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/close.png</normaloff>:/img/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭(O)</string>
   </property>
   <property name="statusTip">
    <string>关闭活动子窗口</string>
   </property>
  </action>
  <action name="closeallaction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/close-square.png</normaloff>:/img/close-square.png</iconset>
   </property>
   <property name="text">
    <string>关闭所有(A)</string>
   </property>
   <property name="statusTip">
    <string>关闭所有子窗口</string>
   </property>
  </action>
  <action name="casecadeAction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/Center.png</normaloff>:/img/Center.png</iconset>
   </property>
   <property name="text">
    <string>层叠(C)</string>
   </property>
   <property name="statusTip">
    <string>层叠子窗口</string>
   </property>
  </action>
  <action name="Titleaction">
   <property name="icon">
    <iconset>
     <normaloff>:/img/android-chrome-512x512.png</normaloff>:/img/android-chrome-512x512.png</iconset>
   </property>
   <property name="text">
    <string>平铺(T)</string>
   </property>
  </action>
  <action name="previousaction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/behind.png</normaloff>:/img/behind.png</iconset>
   </property>
   <property name="text">
    <string>前一个(V)</string>
   </property>
   <property name="statusTip">
    <string>移动焦点到前一个子窗口</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="nextaction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/after.png</normaloff>:/img/after.png</iconset>
   </property>
   <property name="text">
    <string>下一个(X)</string>
   </property>
   <property name="statusTip">
    <string>后一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="aboutaction">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/about.png</normaloff>:/img/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(A)</string>
   </property>
   <property name="statusTip">
    <string>关于LuWPS的信息</string>
   </property>
  </action>
  <action name="underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/underline.png</normaloff>:/img/underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
